server.js

```diff
@@ -13,10 +13,21 @@ server.get("/", (req, res) => {
 });
 
 server.post("/", bodyParser, (req, res) => {
-  const { name, message } = req.body;
-  const created = Date.now();
-  messages.push({ name, message, created });
-  res.redirect("/");
+  const name = req.body.name;
+  const message = req.body.message;
+
+  const errors = {};
+  if(!name){
```

Comment:
please add a vliadation for message too here

Reply:
resolved